src/pages/nonprofit/issues/index.tsx

```diff
@@ -1,9 +1,184 @@
-import { Flex } from "@chakra-ui/react";
+import {
+  Button,
+  Flex,
+  Heading,
+  Text,
+  VStack,
+  Tabs,
+  Tab,
+  TabList,
+  TabPanels,
+  TabPanel,
+} from "@chakra-ui/react";
+import Link from "next/link";
+import { useEffect, useState } from "react";
+import { nonprofitGetIssues, nonprofitGetProjects } from "src/actions/Project";
+import IssuesTable from "src/components/nonprofit/issues/IssuesTable";
+import MaintenanceProjectCard from "src/components/nonprofit/issues/MaintenanceProjectCard";
+import { dateDiffInDays, dateToMMDDYYYY } from "src/utils/dates";
+import { showError } from "src/utils/notifications";
+import {
+  Issue,
+  IssueStatus,
+  Chapter,
+  Project,
+  ProjectStage,
+} from "src/utils/types";
+import urls from "src/utils/urls";
 
 function NonprofitIssuesPage() {
+  const [projects, setProjects] = useState<Project[]>();
+  const [currProject, setCurrProject] = useState<Project>();
+  const [issues, setIssues] = useState<Issue[]>();
+
+  useEffect(() => {
+    async function fetchProjects() {
+      let projects: Project[] = await nonprofitGetProjects({
+        status: ProjectStage.MAINTENANCE,
+      });
+
+      projects = projects.map((project) => {
```

Comment:
After changing the Project model to not include maintenanceEnd and remainingDays, you will get type errors here.  To handle this, you can create a type that is the union of a project and your additional properties maintenanceEnd/remainingDays, and store an array of that type in the state instead.  That way the Project model still matches the schema and you can extend it to do what you need to do in this component.